dev-libs/libgpg-error/files/libgpg-error-1.27-no-undefined.patch

```diff
@@ -0,0 +1,13 @@
+https://lists.gnupg.org/pipermail/gnupg-devel/2017-March/032671.html
+
+--- libgpg-error-1.27/src/Makefile.in
++++ libgpg-error-1.27/src/Makefile.in
```

Comment:
should change Makefile.am and not Makefile.in + add autoreconf

However, I am almost sure that in the bug we already discussed that this should be pushed against upstream.